ui/src/app/job-details/tabs/tabs.component.ts

```diff
@@ -87,6 +87,18 @@ export class TaskDetailsComponent implements OnInit, OnChanges {
     return false;
   }
 
+  hasFailures(): boolean {
+    return this.job.failures && (this.job.failures.length > 1)
+  }
+
+  hasInputs(): boolean {
+    return this.job.inputs && (Object.keys(this.job.inputs).length > 1)
```

Comment:
this was `Object.keys(this.job.inputs).length !== 0` before, I think there are no differences between them based on my understanding right?

Reply:
changed